static/admin/config.yml

```diff
@@ -8,22 +8,20 @@ publish_mode: editorial_workflow
 media_folder: "src/images" # where media files will be stored in the repo
 
 collections:
-  - name: "blog" # Used in routes, e.g., /admin/collections/blog
-    label: "Blog" # Used in the UI
-    folder: "src/pages/blog" # The path to the folder where the documents are stored
-    create: true # Allow users to create new documents in this collection
-    slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
-    fields: # The fields for each document, usually in front matter
-      - {label: "Layout", name: "layout", widget: "hidden", default: "blog"}
-      - {label: "Path", name: "path", widget: "string"}
-      - {label: "Date", name: "date", widget: "datetime"}
-      - {label: "Title", name: "title", widget: "string"}
-      - {label: "Body", name: "body", widget: "markdown"}
   - name: "project"
     label: "Project"
     folder: "src/pages/project"
     create: "true"
     slug: "{{title}}"
     fields:
       - {label: "Title", name: "title", widget: "string"}
-      - {label: "Body", name: "body", widget: "markdown"}
+      - {label: "Summary Description", name: "summary", widget: "markdown"}
+      - {label: "Stage", name: "stage", widget: "select", options: ["Proposed", "Planning", "Implementing", "Live"]}
```

Comment:
Here is where I spun some cycles trying to learn how to create options from select dropdown.